README.md

```diff
@@ -119,10 +122,10 @@ func main() {
 	fmt.Println("Successfully put file with name:", name)
 
 	// GET
-	name := "hello.txt"
+	name = "art.svg"
 	f, err := drive.Get(name)
 	if err != nil {
-		fmt.Println("Failed to get file with name:", name)
+		fmt.Println("Failed to get file with name:", name, err)
```

Comment:
`err` after name is confusing, we can just do `fmt.Println("Failed to get file:", err)`